public/config.js

```diff
@@ -18,6 +43,7 @@ System.config({
     "composer-components": "github:guardian/tools-components@1.0.8",
     "core-js": "npm:core-js@1.1.1",
     "css": "github:systemjs/plugin-css@0.1.13",
+    "localforage": "npm:localforage@1.2.10",
     "github:angular/bower-angular-route@1.4.3": {
```

Comment:
Noticed that angular is being included here. Are we using Angular in viewer? Can't seem to find it.